api/data/tree.go

```diff
@@ -75,9 +75,62 @@ func (p *PartInfo) ToHeaderString() string {
 
 // LockInfo is lock information to create appropriate tree node.
 type LockInfo struct {
-	ID           uint64
-	LegalHoldOID *oid.ID
-	RetentionOID *oid.ID
-	UntilDate    string
-	IsCompliance bool
+	id uint64
+
+	legalHoldOID oid.ID
+	setLegalHold bool
+
+	retentionOID oid.ID
+	setRetention bool
```

Comment:
I thought you decided to group fields like
```
id

legalHoldOID
setLegalHoldOID

retentionOID
setRetention

untilDate
isCompliance
```

Shouldn't be empty line placed after `setRetention` or line 82 removed?

Reply:
Actually not. I wanted to group fields such way to show that `untilDate` and `isCompliance` are tough related to `retentionOID`